tests/acceptance/AccessoriesCept.php

```diff
@@ -5,20 +5,20 @@
 $I->am('logged in user');
 $I->wantTo('ensure that the accessories listing page loads without errors');
 $I->lookForwardTo('seeing it load without errors');
-$I->amOnPage('/admin/accessories');
+$I->amOnPage('/accessories');
```

Comment:
Should probably eventually switch these to routes, since that's what we're using mostly now in the app